src/main/java/de/novatec/marioai/tools/MarioAiRunner.java

```diff
@@ -46,86 +47,53 @@
+		exitOnFinish = marioAiRunnerBuilder.isExitOnFinish();
+		pushMetrics = marioAiRunnerBuilder.isPushMetrics();
+		
 	}
 	
 	/**
-	 *  Equals the call of @link #run(Agent, LevelConfig, boolean, int)} with attributes run(agent, levelConfig, randomize, 3).
-	 * @param agent Agent to be evaluated
-	 * @param levelConfig Configuration for the {@link}LevelGenerator. Either use preset levelConfigs or create your own
-	 * @param randomize determines if the level should be randomized based on the levelConfig
-	 */
-	public static void run(Agent agent,LevelConfig levelConfig,boolean randomize) {		
-		run(agent, levelConfig, 24,3,randomize, true,false);
-	}
-	/**
-	 *  Equals the call of @link #run(Agent, LevelConfig, boolean, int)} with attributes run(agent, levelConfig, randomize, 3).
-	 * @param agent Agent to be evaluated
-	 * @param levelConfig Configuration for the {@link}LevelGenerator. Either use preset levelConfigs or create your own
-	 */
-	public static void run(Agent agent,LevelConfig levelConfig) {
-		run(agent, levelConfig, 24,3, false, true,false);
-	}
-
-	/**
-	 *  Hides some attributes of @link #run(Agent, LevelConfig, int,int, boolean, boolean, boolean). Standard values will be used for missing attributes.
-	 * @param agent Agent to be evaluated
-	 * @param levelConfig Configuration for the {@link}LevelGenerator. Either use preset levelConfigs or create your own
-	 * @param randomize determines if the level should be randomized based on the levelConfig
-	 * @param zoomFactor multiplier for window height/width (Standard: 320x240)
+	 * 
+	 * @return List<EvaluationInfo> runs all agents using the current settings(variables) and returns the results
 	 */
-	public static void run(Agent agent,LevelConfig levelConfig,boolean randomize, int zoomFactor) {		
-		run(agent, levelConfig, 24,zoomFactor,randomize, true,false);
-	}
-	
-	public static List<EvaluationInfo> multiAgentRun(List<Agent> agents, LevelConfig levelConfig,Task task,int fps,int zoomFactor, boolean randomize, boolean viewable, boolean debugView, boolean exitOnFinish, boolean pushMetrics) {
-		
-		//log.info("MarioAi - trying to start evaluation...");
-		if(agents==null){
-			log.error("Agents List can't be null!\nPlease use a proper List with agents! - no evaluation started");
-			return new ArrayList<>();
-		}
-		if(agents.isEmpty()) {
-			log.error("Agents List is empty - no evaluation started");
-			return new ArrayList<>();
-		}
-		if(levelConfig==null&&!randomize) log.warn("LevelConfig is null, Level will be randomized!");
-		if(randomize||levelConfig==null) levelConfig=LevelConfig.randomize(levelConfig);
-		if(task==null) {
-			log.error("Task can't be null");
-			log.error("Exiting...");
-			return new ArrayList<>();
-		}
-		if(zoomFactor<1) zoomFactor=1;	
+	public static List<EvaluationInfo> run() {
 		
 		RunnerOptions baseOptions=new RunnerOptions(agents.get(0),levelConfig,task);
-
+		if(randomize) levelConfig = LevelConfig.randomize(levelConfig);
```

Comment:
Möglicherweise wäre es lesbarer und weniger fehleranfällig wenn selbst bei einzeiligen Statement in einem if- oder Schleifenkörper die geschweiften Klammern genutzt werden.